src/main/java/org/mifos/processor/bulk/utility/TransactionParser.java

```diff
@@ -0,0 +1,66 @@
+package org.mifos.processor.bulk.utility;
+
+import org.mifos.processor.bulk.schema.Transaction;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class TransactionParser {
+
+    private static final Logger logger = LoggerFactory.getLogger(TransactionParser.class);
+
+    private TransactionParser() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    public static Transaction parseLineToTransaction(String line) {
+        try {
+            String[] parts = line.split(",", -1);
+            Transaction transaction = new Transaction();
+
+            if (parts.length > 0 && !parts[0].isEmpty()) {
```

Comment:
check if there is a better way to ready all the content of CSV using csv schema

Reply:
We are not using any library to parse directly into the object. This is the only way to parse it in a reliable way.